spec/models/resource_spec.rb

```diff
@@ -18,4 +18,39 @@
   end
 
   it_behaves_like 'indexable', :resource
+
+  describe 'privacy' do
+    let(:resource) { create(:resource, privacy: :priv) }
+
+    describe 'after_commit on: [:create]' do
+      it "adds the #{name} to the search index after creation" do
+        allow(AddToIndexJob).to receive(:perform_async)
+
+        expect(AddToIndexJob).not_to have_received(:perform_async).
+          with(resource.class.name, resource.id)
+      end
+    end
+
+    describe 'after_commit on: [:update]' do
+      it "updates the #{name} index in the search index after update" do
```

Comment:
Same as above. Can you update this spec name to be `"does not update the #{name} to the search index after update"`?